<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JoinGroupChat</class>
 <widget class="QDialog" name="JoinGroupChat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Join groupchat</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="viewPage">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Select account:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="accountBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="joinPage">
      <layout class="QVBoxLayout" name="joinLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="saveBookmark">
         <property name="text">
          <string>Save to bookmarks</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="bookmarkName">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="bookmarksPage">
      <layout class="QVBoxLayout" name="bookmarksLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Select bookmark:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="bookmarksBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="ActionBox" name="actionBox" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ActionBox</class>
   <extends>QWidget</extends>
   <header>actionbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>saveBookmark</sender>
   <signal>toggled(bool)</signal>
   <receiver>bookmarkName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
